### ISSUE TITLE
About load balancing

### ISSUE THREAD

**Issue by wwzll123** [Clin0212/HydraLoRA#28]
Thanks for the great study!

K-means clustering seems to be used only to determine the number of experts. Are training samples forcibly assigned to specific experts during fine-tuning? Or are there additional loss for load balancing? The common approach is Top-K strategy, but this paper does not adopt it. I didn't find a description of load balancing in the paper.

Looking forward to your reply. Thanks!
